rero_ils/modules/acq_orders/permissions.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""AcqOrder permissions."""
+
+from ...permissions import staffer_is_authenticated
+
+
+def can_read_update_delete_acq_order_factory(record, *args, **kwargs):
```

Comment:
this seems can be applied to other resource. Can you change the name for a more general rule? You can add it to the main permissions file.

Reply:
ok